0001-force-breeze-icon-theme.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/src/main.cpp b/src/main.cpp
+index e326a44..5d23485 100644
+--- a/src/main.cpp
++++ b/src/main.cpp
+@@ -12,6 +12,7 @@
+ #include <QQuickStyle>
+ #include <QUrl>
+ #include <QtQml>
++#include <QIcon>
+ 
+ #ifdef Q_OS_ANDROID
+ #include <QGuiApplication>
+@@ -48,6 +49,7 @@ Q_DECL_EXPORT int main(int argc, char *argv[])
+     if(qEnvironmentVariableIsEmpty("QT_QUICK_CONTROLS_STYLE")) {
+         QQuickStyle::setStyle(QStringLiteral("org.kde.desktop"));
+     }
++    QIcon::setThemeName(QStringLiteral("breeze"));
```

Comment:
I don't think forcing is a good idea, fallbacking as i suggested seems nicer for users.

Reply:
I'm pretty sure fallbacking only occurs when it doesn't detect any theme at all (it still detects Adwaita so it won't fallback)

At least when I tried, I still ended up without the necessary icons :thinking:

Reply:
You're [partially] right, it doesn't fallback from adwaita to breeze, it's what i wanted it to do when i added this feature, but i seem to have botched the code, meh